nixos/modules/config/nsswitch.nix

```diff
@@ -10,35 +10,44 @@ let
   canLoadExternalModules = config.services.nscd.enable;
   myhostname = canLoadExternalModules;
   mymachines = canLoadExternalModules;
+  # XXX Move these to their respective modules
```

Comment:
Could these also be moved into the individual systems.

We have a similar mess inside the pam module, and it'd be nice to clean all this up.

Reply:
Yeah, that's what I wanted to say

Reply:
Can you add a commit on top that does this, for some of the NSS modules?

I'd assume the systemd and oslogin nss modules should be a somewhat trivial example, and then we could create a follow-up ticket tracking the migration of the others.

Reply:
Just added the commit. I used sss instead, because it allows me to drop the `servicesArray` ;)